### ISSUE TITLE
CRIT Mod: "Dig a pit" visible in construction menu

### ISSUE THREAD

**Issue by BaumgeistOne** [CleverRaven/Cataclysm-DDA#33647]
# Describe the bug

Using the CRIT Mod, the option "dig a pit" is visible in the construction menu. Using it does not yield soil.

<s>Digging a pit via the action menu yields soil and other stuff (but you don't need soil to fill it back up, see separate issue #29303).

Digging a pit via the construction menu does not yield anything.</s>

# Steps To Reproduce

1. Spawn a shovel.
2. Dig a pit via the construction menu.
3. Notice the lack of material.

# Expected behavior

I would expect the same result, no matter which menu I use.

# Versions and configuration

- OS: Windows
    - OS Version: MINGW/CYGWIN/MSYS2 on Windows 7
- Game Version: 0.D-7053-gdfe287e [64-bit]
- Graphics Version: Tiles
- Mods loaded: [
    Dark Days Ahead [dda],
    Aftershock [aftershock],
    C.R.I.T Expansion Mod [crt_expansion],
    DeadLeaves' Fictional Guns [FIC_Weapons],
    Garden Pots [growable-pots],
    Icecoon's Arsenal [ew_pack],
    Makeshift Items Mod [makeshift],
    Medieval and Historic Classes and Shields [Medieval_Stuff],
    More Survival Tools [More_Survival_Tools],
    Mythological Replicas [nw_pack],
    Modular Turrets [modular_turrets],
    Salvaged Robots [Salvaged_Robots],
    Alternative Map Key [alt_map_key],
    Hydroponics [hydroponics],
    Mutant NPCs [mutant_npcs],
    Mythical Martial Arts [MMA],
    Beta National Guard Camp [national_guard_camp],
    More Locations [more_locations],
    Fuji's More Buildings [FujiStruct],
    Urban Development [Urban_Development],
    Folding Parts pack [deoxymod],
    Vehicle Additions Pack [blazemod],
    Tanks and Other Vehicles [Tanks],
    sees-player icon, +attitude [sees_player],
    StatsThroughSkills [StatsThroughSkills]
]

# Additional context

Since #31209, "dig a pit" is not visible anymore in the basegame.

<s>In my opinion, digging a pit should be removed from the construction menu.</s>

**Comment by alkemann** [comment]
I believe there are already PRs to this effect. See #33638

**Comment by alkemann** [comment]
There is also the claim that it is fixed in #31209. Can you update and verify @BaumgeistOne

**Comment by BaumgeistOne** [comment]
The build I tested this on was from yesterday. I just updated the game (build 9564) and tested this again, and it's still possible for me to select "dig a pit" from the construction menu.

Maybe one of the mods is interfering with this? But as far as I know, every one of those comes with the game.
![CataclysmDigBug](https://user-images.githubusercontent.com/53833479/63962453-6e80ba00-ca82-11e9-868d-eb7f018b9788.png)
